File: mog-rs/factory/mysql-tables-to-go/tests/expected.go
type Actor struct {
	ActorId int32 `db:"actor_id"`
	FirstName string `db:"first_name"`
	LastName string `db:"last_name"`
	LastUpdate time.Time `db:"last_update"`
}

type Film struct {
	FilmId int32 `db:"film_id"`
	Title string `db:"title"`
	Description *string `db:"description"`
	ReleaseYear *int16 `db:"release_year"`
	LanguageId int32 `db:"language_id"`
	OriginalLanguageId *int32 `db:"original_language_id"`
	RentalDuration int8 `db:"rental_duration"`
	RentalRate string `db:"rental_rate"` // TODO(mog): MySQL exact-precision column; mapped to a Go string (no native fixed-point) -- parse with a fixed-point library for arithmetic.
	Length *int16 `db:"length"`
	ReplacementCost string `db:"replacement_cost"` // TODO(mog): MySQL exact-precision column; mapped to a Go string (no native fixed-point) -- parse with a fixed-point library for arithmetic.
	Rating *string `db:"rating"` // TODO(mog): MySQL enum; represented as a Go string (define named constants if you need type safety).
	SpecialFeatures *string `db:"special_features"` // TODO(mog): MySQL set; comma-joined values in one Go string -- split on ',' or model as a slice.
	LastUpdate time.Time `db:"last_update"`
}

type Staff struct {
	StaffId int32 `db:"staff_id"`
	FirstName string `db:"first_name"`
	LastName string `db:"last_name"`
	AddressId int32 `db:"address_id"`
	Picture []byte `db:"picture"`
	Email *string `db:"email"`
	StoreId int32 `db:"store_id"`
	Active bool `db:"active"`
	Username string `db:"username"`
	Password *string `db:"password"`
	LastUpdate time.Time `db:"last_update"`
}

type Customer struct {
	CustomerId int32 `db:"customer_id"`
	StoreId int32 `db:"store_id"`
	FirstName string `db:"first_name"`
	LastName string `db:"last_name"`
	Email *string `db:"email"`
	AddressId int32 `db:"address_id"`
	Active bool `db:"active"`
	CreateDate time.Time `db:"create_date"`
	LastUpdate *time.Time `db:"last_update"`
}

File: mog-rs/factory/mysql-tables-to-csharp/tests/expected.cs
public class Actor
{
    public int ActorId { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public DateTime LastUpdate { get; set; }
}

public class Film
{
    public int FilmId { get; set; }
    public string Title { get; set; }
    public string? Description { get; set; }
    public int? ReleaseYear { get; set; }
    public int LanguageId { get; set; }
    public int? OriginalLanguageId { get; set; }
    public sbyte RentalDuration { get; set; }
    public decimal RentalRate { get; set; }
    public short? Length { get; set; }
    public decimal ReplacementCost { get; set; }
    public string? Rating { get; set; } // TODO(mog): MySQL enum; values lost, consider a C# enum instead of string.
    public string? SpecialFeatures { get; set; } // TODO(mog): MySQL set; multi-value set lost, stored as string.
    public DateTime LastUpdate { get; set; }
}

public class Staff
{
    public int StaffId { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public int AddressId { get; set; }
    public byte[]? Picture { get; set; }
    public string? Email { get; set; }
    public int StoreId { get; set; }
    public bool Active { get; set; }
    public string Username { get; set; }
    public string? Password { get; set; }
    public DateTime LastUpdate { get; set; }
}

public class Customer
{
    public int CustomerId { get; set; }
    public int StoreId { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string? Email { get; set; }
    public int AddressId { get; set; }
    public bool Active { get; set; }
    public DateTime CreateDate { get; set; }
    public DateTime? LastUpdate { get; set; }
}
